CMakeLists.txt

```diff
@@ -5,7 +5,8 @@ project(sycl-blas)
 enable_testing()
 
 set(SYCLBLAS_INCLUDE ${CMAKE_CURRENT_SOURCE_DIR}/include)
-set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/Modules)
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake/Modules)
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/include/computecpp-sdk/cmake/Modules)
 
 include(FindOpenCL)
 include(FindComputeCpp)
```

Comment:
Nope - I had some cmake issues (unrelated to the FindComputeCpp), so I reverted it in case that was the issue. I think using find_package() would probably be best as well, so I'll go back to that.